doc/classes/Camera2D.xml

```diff
@@ -346,16 +352,21 @@
 		<member name="editor_draw_screen" type="bool" setter="set_screen_drawing_enabled" getter="is_screen_drawing_enabled">
 		</member>
 		<member name="limit_bottom" type="int" setter="set_limit" getter="get_limit">
+			Bottom limit the camera will be able to move in pixels.
 		</member>
 		<member name="limit_left" type="int" setter="set_limit" getter="get_limit">
+			Left limit the camera will be able to move in pixels.
```

Comment:
Clarify wording (applies to all four directions):

    Left scroll limit in pixels. The camera stops moving when reaching this value.